src/config.ts

```diff
@@ -97,8 +131,10 @@ export const rollupChain = defineChain({
 //   }
 // );
 
+
+
 export const parentClient = createPublicClient({
-  chain: baseSepolia,
+  chain: JibchainTestnet,
```

Comment:
This is the wrong approach. We'd want to make the parent chain configurable from environment variables so that there's no need to change names and commit different configurations to support new chains.